### PR TITLE
feat:theme color changes,icon added,image added,login ,logout  changes ,text changes

### PR DESCRIPTION
**Pull request by ekram2d**
#Theme color ,image icon ,text changes  ,all features are implemets 

Booking Dei! - Home Page (1)

### CODE REVIEW

**Comment by annur-nifty** [comment]
Good, try to customize other things as well

**Comment by ekram2d** [comment]
> Good, try to customize other things as well

ok vai .

**Comment by ekram2d** [comment]
#Discover The Ease Of Booking Dei card added 
image

File: components/CompanyLogo/CompanyLogo.js
Comment (annur-nifty) [inline components/CompanyLogo/CompanyLogo.js:1]: Awesome

**Comment by ekram2d** [comment]
-Instant Room Insights section added 
image
-mobile version 
image

**Comment by ekram2d** [comment]
-Easy Profile Acces added
image

**Comment by ekram2d** [comment]
Tab system added .Image and text change with clicking tab
image

**Comment by ekram2d** [comment]
-Full website landing page 
Booking Dei! - Home Page

**Comment by ekram2d** [comment]
> #Theme color ,image icon ,text changes ,all features are implemets
> 
> Booking Dei! - Home Page (1)

@annur-nifty check vai

**Review by annur-nifty — COMMENTED** [review]
good

**Pull request closed without merging**